tests/functional/centos/7/mon-osd/hosts

```diff
@@ -0,0 +1,5 @@
+[mons]
+mon0 monitor_interface=enp0s8
```

Comment:
There probably is, I can look into it. It'd be nice to comment the tox.ini as well with some details on each scenario.